### PR TITLE
DEVDOCS-2081: Dev Center | Theme Docs | Catalog Price Object Examples

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2081](https://jira.bigcommerce.com/browse/DEVDOCS-2081)

## What changed?
* applied sentence case
* removed outdated images
* added new image
* updated links
* removed capitalization of control panel and catalog
* changed table formatting from html to markdown
* put UI element names in bold

### CODE REVIEW

File: docs/stencil-docs/developing-further/catalog-price-object.md [hunk 1]
```diff
@@ -111,42 +83,31 @@ lineNumbers: true
...
 
-Items to note about the Catalog Price Object above:
+In this example:
 
-* `without_tax` -- this property maps to the control panel’s **Default Price** field.
-* `rrp_without_tax` -- `rrp` is an abbreviation for "regular retail price." This property maps to the control panel's **MSRP (Retail Price)** field.
-* `saved` --  this property is the difference between the existing values for `without_tax` (**Default Price**) and `rrp_without_tax` (**MSRP**).
+* `without_tax` - this property maps to the control panel’s **Default Price** field.
+* `rrp_without_tax` - `rrp` stands for "regular retail price." This property maps to the control panel's **MSRP** field.
+* `saved` -  this property is the difference between the existing values for `without_tax` (**Default Price**) and `rrp_without_tax` (**MSRP**).
```
Comment (aglensmith) [inline docs/stencil-docs/developing-further/catalog-price-object.md:90]: these properties would look better in a table.

File: docs/stencil-docs/developing-further/catalog-price-object.md [hunk 2]
```diff
@@ -193,79 +154,75 @@ lineNumbers: true
...
-     ... [code to display on-sale strikeout pricing or content for an on sale product]  ...
+     ... [code to display on-sale strikeout pricing or content for a discounted product]  ...
   {{/or}}
 ```
 
-## Catalog Price Object Examples Including and Excluding Tax
+## Catalog price object examples including and excluding tax
 
-Depending on how the store has been set up in the Control Panel’s **Store Setup** > **Tax** > **Configure Tax Display Settings**, the *Catalog Price Object* properties may represent prices in the following ways:
+Depending on how the store has been set up in the control panel’s **Store Setup** > **Tax** > **Tax Settings** > **Configure Tax Display Settings**, the `price` object will display prices in one of the following ways:
```
Comment (aglensmith) [inline docs/stencil-docs/developing-further/catalog-price-object.md:192]: You can hyper link the navigation path using `https://login.bigcommerce.com/deep-links/manage/` -- when someone navigates to the url, they'll login to their control panel, then be taken to the control panel page specified by the part of the url after `manage`.

File: docs/stencil-docs/developing-further/catalog-price-object.md [hunk 3]
```diff
@@ -298,34 +255,34 @@ lineNumbers: true
 }
 ```
 
-Items to note about the *Catalog Price Object* example above:
+In this example:
 
-* `with_tax`: This property represents the `without_tax` value plus a 10% tax markup.
-* `rrp_with_tax`: This property represents the `rrp_without_tax` value plus a 10% tax markup.
+* `with_tax` - this property represents the `without_tax` value plus a 10% tax markup.
+* `rrp_with_tax` - this property represents the `rrp_without_tax` value plus a 10% tax markup.
```
Comment (aglensmith) [inline docs/stencil-docs/developing-further/catalog-price-object.md:261]: might look better as a table

File: docs/stencil-docs/developing-further/catalog-price-object.md (hunk 2 above)
Comment (ektravel) [reply docs/stencil-docs/developing-further/catalog-price-object.md:192]:
```suggestion
Depending on the store’s  [**Tax Display Settings**](https://login.bigcommerce.com/deep-links/manage/settings/tax/settings), the `price` object will display prices in one of the following ways:

```

File: docs/stencil-docs/developing-further/catalog-price-object.md (hunk 3 above)
Comment (ektravel) [reply docs/stencil-docs/developing-further/catalog-price-object.md:261]:
```suggestion
* `rrp_with_tax` - this property represents the `rrp_without_tax` value plus a 10% tax markup.

```

File: docs/stencil-docs/developing-further/catalog-price-object.md [hunk 4]
```diff
@@ -379,73 +342,33 @@ lineNumbers: true
...
+| `sale_price_with_tax` and `sale_price_without_tax` | These properties represent the product's defined sale price.|
+| `saved` | This value is based on the difference between the `with_tax` and `non_sale_price_with_tax` values.|
+
+## Control panel quick reference
+
+The table below explains the mapping between the control panel fields and the catalog price object properties and values.
+
+| Field | Catalog Price Object Property | Description |
+| ----------- | ----------- | ----------- |
+| Default Price (excluding tax) |<ul><li><b>Sale Price</b> is defined:</li> <code>{{ product.price.non_sale_price_without_tax }}</code><li><b>Sale Price</b> is not defined:</li> <code>{{ product.price.without_tax }}</code></ul>| The standard store price for the product. |
```
Comment (uwikaiddi) [inline docs/stencil-docs/developing-further/catalog-price-object.md:360]: Code isn't appearing for this section

**Pull request merged**